cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -296,49 +296,18 @@ public void performRunLevelStop(HttpRequest request, HttpResponder responder,
                                   @PathParam("app-id") String appId,
                                   @PathParam("program-type") String type,
                                   @PathParam("program-id") String programId,
-                                  @PathParam("run-id") String runId,
-                                  @QueryParam("graceful") String gracefulShutdownSecs) throws Exception {
```

Comment:
Any specific reasons we're removing it now? I think we added it in UI not longtime ago?
https://github.com/cdapio/cdap-ui/pull/418

Reply:
Discussed offline